ui/src/app/store/config/selectors.ts

```diff
@@ -518,6 +518,24 @@ const hardwareSyncInterval = createSelector([all], (configs) =>
   getValueFromName<TimeSpan>(configs, "hardware_sync_interval")
 );
 
+/**
+ * Returns MAAS config for whether the TLS expiration notification is enabled.
+ * @param state - The redux state.
+ * @returns Whether the TLS expiration notification is enabled.
+ */
+const tlsCertExpirationNotificationEnabled = createSelector([all], (configs) =>
+  getValueFromName<boolean>(configs, "tls_cert_expiration_notification_enabled")
+);
+
+/**
+ * Returns MAAS config for the interval in which to show TLS expiration notification.
+ * @param state - The redux state.
+ * @returns The interval in which to show TLS expiration notification.
+ */
+const tlsCertExpirationNotificationInterval = createSelector([all], (configs) =>
+  getValueFromName<Days>(configs, "tls_cert_expiration_notification_interval")
```

Comment:
I think the `Days` type alias here really helps to understand what's being returned.